### DISCUSSION TITLE
Overwriting images with the same name

### DISCUSSION THREAD

**Discussion by ALC-finsa** [requarks/wiki#5599] [Error / Bug Report]
Hello, we have just seen that if we have an image added to a page with a name (example image1.jpg), and creating another page you upload a new image but with the name repeated (image1.jpg), it overwrites the original image1.jpg, changing said image on the original page without realizing it.

Couldn't it be warned when uploading images with duplicate names to warn of this and not lead to inadvertently replacing images?

Thank you

**Comment by KainanSu** [comment]
+1, same problem

**Reply by vladyxa13** [reply]
same problem, when u use software for screenshots like LightShot and copy it to buffer and paste
